--- xfce4-appfinder-4.1.99.3/configure.ac.orig	2004-12-21 10:10:06.000000000 +0100
+++ xfce4-appfinder-4.1.99.3/configure.ac	2004-12-28 10:00:24.231925376 +0100
@@ -24,7 +24,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfce4-appfinder], [be cs de en_GB es es_MX eu fa fi gu he it ko lt nl
-		            pa pt_BR pt_PT ro tr zh_CN])
+		            pa pt_BR pt ro tr zh_CN])
 
 dnl Check for required packages
 BM_DEPEND([LIBXFCEGUI4], [libxfcegui4-1.0], [4.1.99])
