--- xfce4-appfinder-4.2.2/configure.ac.orig	2005-05-15 10:10:56.000000000 +0200
+++ xfce4-appfinder-4.2.2/configure.ac	2005-05-19 14:14:42.685401120 +0200
@@ -24,7 +24,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfce4-appfinder], [be bn_IN ca cs da de el en_GB es es_MX et eu
-			    fa fi gu he hu it ja ko lt mr nl pa pt_BR pt_PT
+			    fa fi gu he hu it ja ko lt mr nl pa pt_BR pt
 			    ro ru sk tr uk zh_CN])
 
 dnl Check for required packages
