diff -urN xfce4-appfinder-4.2.2/src/xfce4-appfinder.desktop fix/xfce4-appfinder-4.2.2/src/xfce4-appfinder.desktop
--- xfce4-appfinder-4.2.2/src/xfce4-appfinder.desktop	2005-05-15 10:10:51.000000000 +0200
+++ fix/xfce4-appfinder-4.2.2/src/xfce4-appfinder.desktop	2005-05-25 15:36:32.000000000 +0200
@@ -32,7 +32,7 @@
 Icon=xfce4-appfinder
 Terminal=false
 Type=Application
-Categories=Utility
+Categories=Utility;
 GenericName=Appfinder
 GenericName[et]=Appfinder
 GenericName[sk]=Appfinder
