--- xfce4-appfinder-4.2.1/configure.ac	2005-03-12 15:19:16.000000000 +0100
+++ fix/xfce4-appfinder-4.2.1/configure.ac	2005-03-16 03:08:53.477189928 +0100
@@ -24,7 +24,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfce4-appfinder], [be cs da de en_GB es es_MX et eu fa fi gu
-                            he hu it ja ko lt nl pa pt_BR pt_PT ro ru 
+                            he hu it ja ko lt nl pa pt_BR pt ro ru 
 			    sk tr uk zh_CN])
 
 dnl Check for required packages
