--- xfce4-appfinder-4.2.3/src/xfce4-appfinder.desktop~	2006-04-20 10:18:21.000000000 +0200
+++ xfce4-appfinder-4.2.3/src/xfce4-appfinder.desktop	2006-04-20 10:24:00.000000000 +0200
@@ -27,6 +27,7 @@
 Comment[ko]=응용프로그램 검색
 Comment[lt]=Išvardina įdiegtas programas
 Comment[nl]=Zoek geïnstalleerde programma's
+Comment[pl]=Wyszukiwanie programów w Xfce 4
 Comment[ro]=Căutare aplicaţii instalate
 Comment[ru]=Список установленных приложений
 Comment[sk]=Zoznam nainštalovaných aplikácií
@@ -46,9 +47,9 @@
 GenericName[ko]=응용프로그램 검색
 GenericName[lt]=AppFinder
 GenericName[nl]=Appfinder
+enericName[pl]=Wyszukiwanie programów
 GenericName[ro]=Appfinder
 GenericName[ru]=Поиск программ
 GenericName[sk]=Appfinder
 GenericName[sv]=Programlista
 GenericName[zh_CN]=程序查找器
-
