--- xfce4-appfinder-4.2.0/configure.ac.orig	2005-01-10 13:56:36.000000000 +0100
+++ xfce4-appfinder-4.2.0/configure.ac	2005-01-17 15:49:55.256221304 +0100
@@ -24,7 +24,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfce4-appfinder], [be cs de en_GB es es_MX et eu fa fi gu he hu it ko
-			    lt nl pa pt_BR pt_PT ro ru tr uk zh_CN])
+			    lt nl pa pt_BR pt ro ru tr uk zh_CN])
 
 dnl Check for required packages
 BM_DEPEND([LIBXFCEGUI4], [libxfcegui4-1.0], [4.2.0])
