--- xfce4-appfinder-4.1.99.2/configure.ac.orig	2004-12-11 15:41:09.000000000 +0100
+++ xfce4-appfinder-4.1.99.2/configure.ac	2004-12-13 23:04:34.642740136 +0100
@@ -24,7 +24,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfce4-appfinder], [be de en_GB es es_MX eu fa fi gu he it ko lt nl
-		            pa pt_PT tr zh_CN])
+		            pa pt tr zh_CN])
 
 dnl Check for required packages
 BM_DEPEND([LIBXFCEGUI4], [libxfcegui4-1.0], [4.1.99])
