--- xfce4-appfinder-4.2.3/configure.ac.orig	2005-11-05 18:04:29.000000000 +0100
+++ xfce4-appfinder-4.2.3/configure.ac	2005-11-13 22:10:34.000000000 +0100
@@ -25,7 +25,7 @@
 dnl Check for i18n support
 BM_I18N([xfce4-appfinder], [af be bn_IN ca cs da de el en_GB es es_MX
 			    et eu fa fi gu he hu hy it ja ko ku lt mr
-			    nl pa pl pt_BR pt_PT ro ru sk sl sv tr uk
+			    nl pa pl pt_BR pt ro ru sk sl sv tr uk
 			    zh_CN zh_TW])
 
 dnl Check for required packages
